### ISSUE TITLE
Multiple readings in reviews

### ISSUE THREAD

**Issue by dbruehlmeier** [davidsansome/tsurukame#420]
When there are multiple readings, it would be helpful if all of them are tested in a review before marking an item as OK. This way, the user needs to actually learn _all_ readings.

**Comment by dbruehlmeier** [comment]
For instance, directly after correctly answering a reading, a hint could be displayed that there is another reading and the same item should appear again for review, but not accept the reading which was already used.

**Comment by UInt2048** [comment]
@dbruehlmeier Are you aware this isn't how normal WK works?
The idea behind this is that if you remember the (kanji) reading that's used 80% of the time, the rest that are used 20% of the time will come through vocab.
If it's about vocab, you should know the readings well enough that if you ever heard any other ways to pronounce it, you should be able to know what word it was.

**Comment by dbruehlmeier** [comment]
@UInt2048 Yes, I am aware of that. Here's the example which triggered this suggestion:

The kanji 力 has two On’yomi readings (りょく and りき): https://www.wanikani.com/kanji/%E5%8A%9B

I only studied the reading りき (perhaps because I'm lazy... and it was the easier one for me to remember), so I really struggled with the vocab 入力 when it came up: https://www.wanikani.com/vocabulary/%E5%85%A5%E5%8A%9B

The only mnemonic here is that it's the same on'yomi as the kanji, which is of course correct, the there are two for 力 and I only knew the "wrong" one...

That's why I wished that the app would have "forced" me to learn both readings of the kanji during the reviews.

Does that make sense? It may very well be a "newbie-problem", but I figured it shouldn't hurt to mention it. I really like your app and I'm trying to make a small contribution to make it better.

**Comment by UInt2048** [comment]
@dbruehlmeier Yes, that makes sense. I can tell you that most other kanji won't be like this. Thanks for the example!

**Comment by dbruehlmeier** [comment]
@UInt2048 OK, good to know! So I guess it would be more of a low priority feature request. Of course, feel free to drop it if you think it's not adding any value.
